bins/revme/src/cmd/statetest/models/mod.rs

```diff
@@ -146,6 +128,29 @@ pub struct Authorization {
     signer: Option<Address>,
 }
 
+impl TestAuthorization {
+    pub fn signature(&self) -> Signature {
+        let parity = self.v != U256::ZERO;
```

Comment:
This is not correct and `v` should stay as a big number.

Reply:
Cast `v` to u64 and construct with `Parity::try_from(v)`, this parity is going to be checked inside Revm.

Reply:
Getting overflow error trying to cast v to u64

Reply:
Cast with `try_from` and use `unwrap_or(u64::MAX)` we don't care how big the number is but just that it is 0 or 1.

Reply:
https://github.com/bluealloy/revm/actions/runs/10821550430/job/30023779288?pr=1757

Reply:
using `u64::MAX` is literally assigned the max value of `u64` after having unwrap err, which is causing the `Parity` to trigger invalid parity error.

Reply:
My bad: It is triggering `InvalidParity(value)` with value being `2`.